projects/1.18/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -105,39 +105,78 @@
+  "gui.draconicevolution.dislocator.add_bottom.info": "将目前的位置\n- 添加至列表底部",
+  "gui.draconicevolution.dislocator.add_top.info": "将目前的位置\n- 添加至列表顶部",
   "gui.draconicevolution.dislocator.delete.info": "删除",
-  "gui.draconicevolution.dislocator.double_click_name": "双击以重命名。",
-  "gui.draconicevolution.dislocator.drag_to_move": "按住并拖动来移动位置。",
-  "gui.draconicevolution.dislocator.edit_lock.info": "Toggle edit lock",
-  "gui.draconicevolution.dislocator.fuel": "燃料：",
-  "gui.draconicevolution.dislocator.fuel.info": "The total number of teleports remaining.",
-  "gui.draconicevolution.dislocator.fuel_add_1.info": "Add 1 Ender Pearl from your inventory.",
-  "gui.draconicevolution.dislocator.fuel_add_16.info": "Add 16 Ender Pearls from your inventory.",
-  "gui.draconicevolution.dislocator.fuel_add_all.info": "Add all Ender Pearls from your inventory.",
-  "gui.draconicevolution.dislocator.mode_blink": "Use Mode:",
-  "gui.draconicevolution.dislocator.mode_tp": "Use Mode:",
-  "gui.draconicevolution.dislocator.must_unlock": "(must be unlocked)",
-  "gui.draconicevolution.dislocator.right_click_mode.info": "Switch between \"Teleport to Selected\" and \"Blink\" on right click.\\nYou can also configure keybindings.",
-  "gui.draconicevolution.dislocator.right_click_tp": "右击传送",
-  "gui.draconicevolution.dislocator.update": "设定这个",
-  "gui.draconicevolution.dislocator.update.info": "Update the selected location to your current coordinates.",
-  "gui.draconicevolution.draconium_chest.autofill_all": "Autofill All",
-  "gui.draconicevolution.draconium_chest.autofill_all.info": "Autofill All",
-  "gui.draconicevolution.draconium_chest.autofill_fill": "Autofill Fill",
-  "gui.draconicevolution.draconium_chest.autofill_fill.info": "Autofill Fill",
-  "gui.draconicevolution.draconium_chest.autofill_lock": "Autofill Lock",
-  "gui.draconicevolution.draconium_chest.autofill_lock.info": "Autofill Lock",
-  "gui.draconicevolution.draconium_chest.autofill_off": "Autofill Off",
-  "gui.draconicevolution.draconium_chest.autofill_off.info": "Autofill Off",
-  "gui.draconicevolution.draconium_chest.color_picker": "更改颜色",
-  "gui.draconicevolution.draconium_chest.color_picker.info": "更改颜色",
-  "gui.draconicevolution.draconium_chest.smelt_energy": "Smelt Energy",
-  "gui.draconicevolution.draconium_chest.smelt_speed": "Smelt Speed",
-  "gui.draconicevolution.energy_net.bound_to_invalid": "工具被绑定到无效的方块上！",
+  "gui.draconicevolution.dislocator.double_click_name": "左键双击以重命名",
+  "gui.draconicevolution.dislocator.drag_to_move": "左键单击并拖动以移动",
+  "gui.draconicevolution.dislocator.edit_lock.info": "开/关编辑锁定",
+  "gui.draconicevolution.dislocator.fuel": "燃料",
+  "gui.draconicevolution.dislocator.fuel.info": "剩余的传送次数",
+  "gui.draconicevolution.dislocator.fuel_add_1.info": "从你的物品栏里添加 1 个末影珍珠。\n末影珍珠用作传送燃料。",
+  "gui.draconicevolution.dislocator.fuel_add_16.info": "从你的物品栏里添加 16 个末影珍珠。\n末影珍珠用作传送燃料。",
+  "gui.draconicevolution.dislocator.fuel_add_all.info": "从你的物品栏里添加所有的末影珍珠。\n末影珍珠用作传送燃料。",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |

`从……添加`不符合中文习惯，但是我的也不见得多好。

```suggestion
  "gui.draconicevolution.dislocator.fuel_add_1.info": "从你的物品栏取1个末影珍珠加入其中。\n末影珍珠用作传送燃料。",
  "gui.draconicevolution.dislocator.fuel_add_16.info": "从你的物品栏里添加 16 个末影珍珠。\n末影珍珠用作传送燃料。",
  "gui.draconicevolution.dislocator.fuel_add_all.info": "从你的物品栏里添加所有的末影珍珠。\n末影珍珠用作传送燃料。",
```

---

已改译为“从你的物品栏里取出 xx 个末影珍珠”，玩家能够从后面的描述中了解末影珍珠的去向，就不需要强调“加入其中”了。